temporal-opentracing/src/main/java/io/temporal/opentracing/internal/DefaultOperationNameAndTagsProvider.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing.internal;
+
+import com.google.common.collect.ImmutableMap;
+import io.temporal.opentracing.OperationNameAndTagsProvider;
+import io.temporal.opentracing.StandardTagNames;
+import io.temporal.opentracing.StartSpanContext;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultOperationNameAndTagsProvider implements OperationNameAndTagsProvider {
+
+  private static final String PREFIX_DELIMITER = ":";
+
+  @Override
+  public String getSpanName(StartSpanContext context) {
+    return context.getOptions().getSpanOperationNamePrefix(context.getSpanOperationType())
```

Comment:
I think that prefixes probably should be a part of the strategy and should be moved here from options. If we want to give full flexibility - prefixes could go away completely in some other strategy.